lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -11,9 +11,9 @@ defmodule ExUnit.Case do
 
   When used, it accepts the following options:
 
-    * :async - configure Elixir to run that specific test case in parallel with
-      others. Must be used for performance when your test cases do not change
-      any global state. It defaults to `false`.
+	* :async - configure this specific test case to able to run in parallel
```

Comment:
the  previous indentation was correct can you please revert it back to use two spaces for introducing the bullet and then four spaces for the text? Thank you!